src/screen/GameScreen.java

```diff
@@ -1,5 +1,6 @@
 package screen;
 
+import java.awt.*;
```

Comment:
This is unnecessary change. please remove it.

Reply:
Fixed unnecessary changes